catalog/core/catalog-core-api/src/main/java/ddf/catalog/data/types/Security.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+
+package ddf.catalog.data.types;
+
+/**
+ * <p>
+ * <b> This code is experimental. While this interface is functional and tested, it may change or be
+ * removed in a future version of the library. </b>
+ * </p>
+ */
+public interface Security {
+
+    /**
+     * {@link ddf.catalog.data.Attribute} name for storing the groups with access to the {@link ddf.catalog.data.Metacard}
```

Comment:
Do we want to adjust this documentation to avoid implying only positive access? Technically access is granted or denied by the policy; these attributes could just as easily represent people and groups who are *not* allowed to access the metacard.